wire/msgtx_test.go

```diff
@@ -36,7 +36,7 @@ func TestTx(t *testing.T) {
 
 	// Ensure max payload is expected value for latest protocol version.
 	// Num addresses (varInt) + max allowed addresses.
```

Comment:
I know this comment was already here, but it is clearly wrong.  Mind updating it while you're here?